README.md

```diff
@@ -247,6 +247,7 @@ Metorikku sends automatically on top of what spark is already sending the follow
 * In streaming: number of processed records in batch
 
 You can also send any information you like to the instrumentation output within a metric.
+by default the last column of the schema will be the field value. all other columns will be tags
```

Comment:
1. please update the example influxdb metric here (remove keycolumn): https://github.com/YotpoLtd/metorikku/blob/master/examples/influxdb/movies_metric.yaml

Write the following:
Add an output like below:
```yaml
output:
- dataFrameName: df
  outputType: Instrumentation
  outputOptions:
    # Optional: define a custom time for the event (works only when writing directly influxDB)
    timeColumn: ts
    # Optional: when defined this column's value will become the metric value, by default the last column is the value column
    valueColumn: val
```
Other columns that are not value or time columns will be merged together as the name of the metric. If writing directly to influxDB these will become tags.